src/navigation/messagesStackNavigation.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import * as React from "react";
+import { createNativeStackNavigator } from "@react-navigation/native-stack";
+
+import Messages from "../components/Messages/Messages";
+
+const Stack = createNativeStackNavigator();
+
+const screenOptions = {
+  headerShown: true
+};
+
+const MessagesStackNavigation = (): React.Node => (
```

Comment:
Can we move this out of its own navigator and into `myObservationsStackNavigation`?

The only way to access messages in iNat mobile is via the MyObservations screen. The user should be able to navigate to Messages from `components/Observations/MessagesIcon`. (Sorry, I probably should have specified this in the Github issue).